src/components/App/index.tsx

```diff
@@ -46,8 +123,9 @@ class App extends React.Component<
 
 const mapStateToProps = (state: ApplicationState): PropsFromState => {
   return {
+    apiState: state.api,
     toggledOn: state.example.toggledOn,
   };
 };
 
-export default connect(mapStateToProps)(App);
+export default connect(mapStateToProps)(AppBase);
```

Comment:
Why are we calling it `AppBase` instead of just `App`?

Reply:
Once we port over our linting rules from frontend you'll find out 😄  There's one in there that doesn't let you name a non-default export identical to the module (iirc). This is a good rule because it's easy to accidentally mix up named and default imports.